README.md

```diff
@@ -2,10 +2,19 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e761d8b0e15b42a092388e490682ae08)](https://www.codacy.com/app/ons-sdc/sdx-seft-consumer-service?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/sdx-seft-consumer-service&amp;utm_campaign=Badge_Grade)
 
-Microservice for consuming SEFT files from RAS and sending to ONS internal
-
 ### License
 
 Copyright ©‎ 2016, Office for National Statistics (https://www.ons.gov.uk)
 
-Released under MIT license, see [LICENSE](LICENSE) for details.
\ No newline at end of file
+Released under MIT license, see [LICENSE](LICENSE) for details.
+
+### Overview
+Microservice for consuming SEFT files from RAS via a Rabbit MQ and sending onto the internal ONS network
+
+# Generate the public/private key pair
```

Comment:
Could do with a proper "Getting started" section and a block covering env vars (like https://github.com/ONSdigital/sdx-receipt-rrm)